frontend_v2/src/app/components/challenge/challengesubmit/challengesubmit.component.html

```diff
@@ -236,6 +236,40 @@ <h5 class="fw-light">Make Submission</h5>
                     </div>
                   </div>
                 </div>
+                <div ng-show="challenge.currentPhaseLeaderboardPublic">
+                  <p class="fs-14 w-400" ng>Select submission visibility:</p>
+                  <li>
+                      <input 
+                        type="radio" 
+                        name="submissionVisibility" 
+                        class="with-gap selectPhase" 
+                        id="isPublicubmission"
+                        value="isPublicubmission"
```

Comment:
we don't need value when we are using label to show the option value right? Can we remove this